docs/my-website/docs/proxy/high_availability_control_plane.md

```
@@ -0,0 +1,190 @@
+import Tabs from '@theme/Tabs';
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)